u4-databases/views/uni-student-details.hbs

```diff
@@ -0,0 +1,10 @@
+  {{#>uni}}
+  <h1>{{ student.code }} - {{ student.name }}</h1>
```

Comment:
คุณสามารถใช้ Partial Context ที่นี่ได้เช่นกัน!
You could use a Partial Context here too!
```js
{{#>uni student}}
<h1>{{ code }} - {{ name }}</h1>
...
```